colabs/wandb_registry/zoo_wandb.ipynb

```diff
@@ -0,0 +1,984 @@
+    "\n",
+    "The target path consists of the name of the organization your team belongs to, the name of the registry, and the name of the collection. There are two ways to get the target path, [interatively with the W&B App UI](https://docs.wandb.ai/guides/registry/link_version#confirm-the-path-of-a-registry-in-the-wb-app-ui) or programmatically with the W&B Python SDK.\n",
+    "\n",
+    "For this example, we will programmatically create the collection target path:\n",
+    "\n",
+    "<!-- #### Interactively get target path of a collection\n",
+    "\n",
+    "1. Navigate to the Registry app at https://wandb.ai/registry/\n",
+    "2. Click on the registry you want to link your artifact version to.\n",
+    "3. At the top of the page, you will see an autogenerated code snippet. Copy the string next to the `target_path` parameter in `run.link_artifact()`. -->\n",
+    "\n",
+    "\n",
+    "#### Programmatically make the collection target path \n",
+    "\n",
+    "The target path of a collectin consists of three parts:\n",
+    "* The name of the organization\n",
+    "* The name of the registry\n",
+    "* The name of the collection within the registry\n",
+    "\n",
+    "If you know these three fields, you can create the full name yourself with string concatanation, f-strings, and so forth:\n",
+    "```python\n",
+    "target_path = f\"{ORG_NAME}/wandb-registry-{REGISTRY_NAME}/{COLLECTION_NAME}\"\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5f2217d9-39df-4e9a-a715-7067e9e5c731",
+   "metadata": {},
+   "source": [
+    "### Publish dataset to registry\n",
+    "\n",
+    "Let's publish our dataset to the Dataset registry in a collection called \"zoo-dataset-tensors\". To do this, we will \n",
+    "\n",
+    "1. Get or create the target path. For this notebook we will programmatically create the target path\n",
+    "1. Initialize a run\n",
+    "1. Create an Artifact object\n",
+    "2. Add each split dataset as individual files to the artifact object\n",
+    "3. Link the artifact object to the collection with `run.link_artifact()`. Here we specify the target path and the artifact we want to link."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "cbfe847c-9229-4428-873b-bde5d0c654e3",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ORG_NAME = \"smle-registries-bug-bash\"\n",
+    "REGISTRY_NAME = \"Dataset\"\n",
+    "COLLECTION_NAME = \"zoo-dataset-tensors\"\n",
+    "\n",
+    "# Path to link the artifact to a collection\n",
+    "dataset_target_path = f\"{ORG_NAME}/wandb-registry-{REGISTRY_NAME}/{COLLECTION_NAME}\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "742c237c-1e8d-4c2e-8ac4-8221146d876e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "PROJECT = \"zoo_experiment\"\n",
+    "TEAM_ENTITY = \"smle-reg-team-2\"\n",
+    "\n",
+    "run = wandb.init(\n",
```

Comment:
It'd be cleaner to ask the user to input a WANDB_API_KEY upstream to not block them in the middle of running the notebook interactively?